ui/nuxeo-page-provider-display-behavior.js

```diff
@@ -437,10 +437,15 @@ export const PageProviderDisplayBehavior = [
     },
 
     _toggleSelectAll() {
-      if (this.selectAllActive) {
+      if (this._isExcludedItemsNotEmpty && this.selectAllActive) {
         this.clearSelection();
+        this._isExcludedItemsNotEmpty = false;
+      } else if (this._isExcludedItemsNotEmpty === false && this.selectAllActive) {
```

Comment:
this._isExcludedItemsNotEmpty === false can be written as !this._isExcludedItemsNotEmpty instead.